src/operator/OrgOverlay.tsx

```diff
@@ -277,6 +277,41 @@ const OrgOverlay: FC = () => {
                       />
                     </Grid.Column>
                   </Grid.Row>
+                  {limits.timeout && (
```

Comment:
Best here would be `Boolean(limits.timeout)`. Probably doesn't matter for this value though, since I don't think the `timeout` object will be anything but a valid object or `null`. Anyway, here's the case that would occur in, since `logical OR` will return the first non falsy value:

`0 && (<Grid.Row>)` - expect to render `<Grid.Row>`, but instill will render `0`, since `0` is not falsy